docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.5'
+
+services:  
+  ubuntu-cpp:
+    # Usage:
+    #   docker-compose build ubuntu-cpp
+    #   docker-compose run --rm ubuntu-cpp
+    # Parameters:
+    #   ARCH: amd64, arm64v8, s390x, ...
+    #   UBUNTU: 18.04, 20.04
+    image: facebookincubator/velox:amd64-ubuntu-22.04-cpp
```

Comment:
Nit: Curious, Wheres the source of this image - how was it built ?

Reply:
NVM , Looks like its the dockerfile referenced below..

Reply:
this will be used for the tag of the image:
```
$ docker images
REPOSITORY                TAG                                                   IMAGE ID       CREATED          SIZE
facebookincubator/velox   amd64-ubuntu-22.04-cpp                                ef72b5402454   26 minutes ago   1.98GB
```